External/pubchem_shape/Wrap/test_rdshapealign.py

```diff
@@ -58,13 +58,13 @@ def test5_ShapeShapeOverlay(self):
     probeShp = rdShapeAlign.PrepareConformer(self.probe, -1)
     tpl = rdShapeAlign.AlignShapes(refShp, probeShp)
     probeCp = Chem.Mol(self.probe)
-    rdShapeAlign.TransformConformer(refShp.shift, tpl[2], probeShp, probeCp.GetConformer(-1))
+    rdShapeAlign.TransformConformer(refShp.shift, refShp.inertialRot, tpl[2], probeShp, probeCp.GetConformer(-1))
```

Comment:
please add a test for at least a couple of the values in `refShp.inertialRot` to make sure that your getter is working correctly